### DISCUSSION TITLE
FMV - FRAM Model Visualiser - Version 3.0.1

### DISCUSSION THREAD

**Discussion by zerprize** [functionalresonance/FMV_Community_Edition#11] [Announcements]
This is the **standard** version of the FRAM Model Visualiser that is freely available as a Progressive Web App at https://fmv.zerprize.co.nz/.
The contents can be unzipped and placed on a webserver to be delivered as a self-contained browser-wasm (Blazor WebAssembly App).

<hr /><em>This discussion was created from the release <a href='https://github.com/Zerprize-Limited/FMV_Community_Edition/releases/tag/'>FMV - FRAM Model Visualiser - Version 3.0.1</a>.</em>

**Comment by zerprize** [comment]
Integrating the basic FMI - FRAM Model Interpreter into the FMV is now complete with the release of version 3.0.1.  
The FMI functionality was previously only available in paid versions of the FMV software but it is now considered standard functionality that is available in the community edition.

Release 3.0.1 is available in the [repository](https://github.com/Zerprize-Limited/FMV_Community_Edition/releases) and on the [GitHub page](https://zerprize-limited.github.io/FMV_Community_Edition/), and matches the public release on https://fmv.zerprize.co.nz/

This marks an important milestone in porting the previous desktop FMV applications to new web browser based application development that makes accessibility from any operating system so much easier.

**Reply by DHSlater1** [reply]
This really is an important development and advance!
